client/lib/mobile-app/index.js

```diff
@@ -1,8 +1,14 @@
+/**
+ * @jest-environment jsdom
+ */
```

Comment:
```suggestion
```

We can remove this; we would need it in the corresponding test file if you wanted to have `window` declared, but since we've double-checked with the `typeof` check below, that won't be necessary either.